bin/convert-owl.py

```diff
@@ -0,0 +1,113 @@
+def download(url: str, path: str):
+    """Download file at url to local path"""
+    _dir = os.path.dirname(path)
+    if not os.path.exists(_dir):
+        os.mkdir(_dir)
+    if not os.path.exists(path):
+        with open(path, 'wb') as f:
+            response = requests.get(url, verify=False)
+            f.write(response.content)
+
+
+def owl_to_obograph(inpath: str):
+    """Convert OWL to Obograph"""
+    parse_results: robot.ParseResults = robot.convert_to_obograph_local(inpath)
+    graph = parse_results.graph_document.graphs[0]
+    return graph
+
+
+# TODO: When https://github.com/INCATools/ontology-access-kit/pull/374 is merged, can add include_all_predicates
+def obograph_to_fhir(inpath: str, out_dir: str, out_filename: str = None, include_all_predicates=False):
+    """Convert Obograph to FHIR"""
+    converter = OboGraphToFHIRConverter()
+    converter.curie_converter = curies.Converter.from_prefix_map(get_default_prefix_map())
+    gd: GraphDocument = json_loader.load(inpath, target_class=GraphDocument)
+    out_path = os.path.join(out_dir, out_filename)
+    # converter.dump(gd, out_path, include_all_predicates=True)
+    converter.dump(gd, out_path)
+
+
+def owl_to_fhir(
+    input_path_or_url: str, out_dir: str = OUTDIR, out_filename: str = None, predicates_is_a_only=False,
+    retain_intermediaries=False
+):
+    """Run conversion"""
+    # Download if necessary
+    # TODO: determine if is URL or path. if URL, download and set new input_path
+    url = ''
+    input_path = input_path_or_url if not url else 'todo'
+    if url:
+        download(url, input_path)
+
+    # Convert to Obograph
+    obograph = owl_to_obograph(input_path)
+
+    # TODO: save obograph file
+    obograph_path = ''
+
+    # Convert to FHIR
+    if not out_filename:
+        _id = '.'.join(os.path.basename(input_path).split('.')[0:-1])  # removes fie extension
+        out_filename = f'CodeSystem-{_id}.json'
+
+    obograph_to_fhir(
+        inpath=obograph_path,
+        out_dir=out_dir,
+        out_filename=out_filename,
+        include_all_predicates=not predicates_is_a_only)
+
+    # Cleanup
+    if not retain_intermediaries:
+        os.remove(obograph_path)
+    print()
+
+
+def cli():
+    """Command line interface."""
+    package_description = 'Convert OWL to FHIR.'
+    parser = ArgumentParser(description=package_description)
+    parser.add_argument('-i', '--input-path-or-url', required=True, help='URL or [ath to OWL file to convert.')
+    parser.add_argument(
+        '-o', '--out-dir', required=False, default=OUTDIR, help='The directory where results should be saved.')
+    parser.add_argument(
+        '-n', '--out-filename', required=False, help='Filename for the primary file converted, e.g. CodeSystem.')
+    # TODO: When https://github.com/INCATools/ontology-access-kit/pull/374 is merged, can add include_all_predicates
+    # parser.add_argument(
+    #     '-p', '--predicates-is-a-only', action='store_true', default=False,
```

Comment:
For parameterization of this conversion function, I felt it made sense to invert the '--include-all-predicates' flag to `--predicates-is-a-only'` for our purposes, since I think that will work better for our use case. But I can just leave the flag param the same way as we're using in OAK (`--include-all-predicates`).